src/components/HistoryCard/historyCard.tsx

```diff
@@ -1,65 +1,63 @@
 import React from "react";
-import Card from "react-bootstrap/Card";
+import { Accordion, Carousel } from "react-bootstrap";
 
 import {
-  card as cardStyle,
+  title as titleStyle,
+  TBAIcon as TBAIconStyle,
+  iconContainer as iconContainerStyle,
+  carouselItem as carouselItemStyle,
 } from "./historyCard.module.css";
-import { Carousel } from "react-bootstrap";
-
-import icon_tba from "../../images/icon_tba.png"
+import icon_tba from "../../images/icon_tba.png";
 
 interface HistoryCardProps {
-  year: String,
-  name: String,
-  showTBA?: boolean,
-  showImages?: boolean,
-  blurb: String,
-  images?: any
+  year: string;
+  name: string;
+  blurb: string;
+  images: string[];
+  showTBA?: boolean;
 }
 
-const carouselItemStyle = {
-  width: "30%",
-};
-
 const makeCarouselItem = (image: string, index: number) => (
   <Carousel.Item key={index}>
-    <img src={image} style={carouselItemStyle} />
+    <img src={image} className={carouselItemStyle} />
   </Carousel.Item>
 );
 
-const tba_icon_style = {
-  'width': '25px',
-  'border-radius': '50%',
-  'margin-bottom': '3px'
+const TBALink = ({ year }: { year: string }) => {
+  const address = "https://www.thebluealliance.com/team/3807/" + year;
+
+  return (
+    <div className={iconContainerStyle}>
+      <a href={address} target="_blank">
+        <img src={icon_tba} className={TBAIconStyle} />
+      </a>
+    </div>
+  );
 };
 
 const HistoryCard = ({
   year,
   name,
-  showTBA = true,
-  showImages = true,
   blurb,
-  images = []
+  images,
+  showTBA = true,
 }: HistoryCardProps) => {
-
   return (
-    <>
-      <Card className={cardStyle}>
-        <Card.Title>{year}: {name}</Card.Title>
-        {showTBA ?
-          <Card.Link href={"https://www.thebluealliance.com/team/3807/" + year} target="_blank"><img src={icon_tba} style={tba_icon_style} /></Card.Link>
-          :
-          <></>
-        }
-        <Card.Text>{blurb}</Card.Text>
-        {showImages ?
-          <Carousel>{images.map(makeCarouselItem)}</Carousel>
-          :
-          <></>
-        }
-      </Card>
-    </>
+    <Accordion.Item eventKey={year}>
+      <Accordion.Header>
+        <p className={titleStyle}>
+          {year}: {name}
+        </p>
+        {showTBA && <TBALink year={year} />}
```

Comment:
Can you explain this syntax? I think I understand it, but boolean && HTML resulting in HTML doesn't make sense to me

Reply:
The React docs on Conditional Rendering explains this syntax: https://react.dev/learn/conditional-rendering#logical-and-operator-

> A [JavaScript && expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) returns the value of its right side (in our case, the checkmark) if the left side (our condition) is true. But if the condition is false, the whole expression becomes false. React considers false as a “hole” in the JSX tree, just like null or undefined, and doesn’t render anything in its place.

So in plain JavaScript, we'd expect the result of the expression to be either literal `false`, or the JSX Element. React takes it a step further, and converts `false` into a null-render.

-----

Previously, you had a ternary operation --- which is totally fine. It was along the lines of `showTBA ? <...element here...> : <></>`. Since if `showTBA==false` we want to render nothing, I used the `&&` shortcut instead to be more concise.

Happy to make an suggestions that would help make this code more clear.